source/ore/publish.rst

```diff
@@ -0,0 +1,138 @@
+
+.. code-block:: json
+
+    [
+        {
+            "modid": "my-plugin",
+            "name": "MyPlugin",
+            "version": "1.0.0",
+            "description": "My first plugin!",
+            "url": "https://spongepowered.org",
+            "authorList": [
+                "windy",
+                "Zidane",
+                "gabizou"
+            ],
+            "requiredMods": [
+                "bookotd@1.0.0",
+                "ore-test@1.0.0",
+                "worldedit@1.0.0"
+            ],
+            "dependencies": [
+                "bookotd@1.0.0",
+                "ore-test@1.0.0",
+                "worldedit@1.0.0"
+            ]
+        }
+    ]
+
+At the very least, each Ore project *must* have the ``modid``, ``name`` and, ``version`` fields completed.
+
+Signing your Plugin
+~~~~~~~~~~~~~~~~~~~
+
+Due to security concerns, Ore requires each plugin file (ZIP or JAR) to be signed with a
+`Pretty Good Privacy (PGP) <https://en.wikipedia.org/wiki/Pretty_Good_Privacy>`__ public key that is associated with
+the email that you signed up to Sponge with. This digital signature ensures that any file uploaded by a user was
+actually intended to be uploaded by the actual account holder.
+
+.. note::
+
+    A digital signature certifies and timestamps a document. If the document is subsequently modified in any way, a
+    verification of the signature will fail. A digital signature can serve the same purpose as a hand-written signature
+    with the additional benefit of being tamper-resistant. The GnuPG source distribution, for example, is signed so
+    that users can verify that the source code has not been modified since it was packaged.
+
+In order to sign your plugin, you must first download a PGP compliant software program, such as
+`GnuPG (GPG) <https://www.gnupg.org/>`__. GPG is available for all major platforms and the binaries can be downloaded
+`here <https://www.gnupg.org/download/index.html#sec-1-2>`__.
+
+Once you have installed GPG and have generated your first public key, you will need to add the key to your account.
+To do this, you can navigate to your profile and click the key symbol next to your avatar.
+
+.. image:: /images/ore/help_1.png
+    :align: center
+    :alt: PGP public key 1
+
+You will then be prompted to enter your key into a text box. You must be sure to enter the key in it's entirety or
+signature validation will fail.
+
+.. image:: /images/ore/help_2.png
+    :align: center
+    :alt: PGP public key 2
+
+Finally you must sign your built plugin with your *private* key of the public key you uploaded. You only need to sign
+the file that you are uploading: for instance, if you are uploading a ZIP with your plugin JAR included, only the ZIP
+needs to (and should only be) signed. The easiest way to sign the file you intend to upload is with the following GPG
+command.
+
+.. code-block:: bash
+
+    gpg --output MyPlugin.jar --sign MyPlugin.jar
```

Comment:
Are you sure this is correct?
On windows I cannot use the file that I would like to sign to output it.

````
> gpg --output Test --sign Test
Sie benötigen eine Passphrase, um den geheimen Schlüssel zu entsperren.
...
Datei 'Test' existiert bereits. Überschreiben (j/N)? j
gpg: 'Test' kann nicht erzeugt werden: No error
gpg: signing failed: Systemfehler ohne gesetzten Systemfehlercode
````

omiting the output parameter or using a different extension such as .gpg solves this issue,